ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/jsonrpc/response/JsonRpcSuccessResponse.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 ConsenSys AG.
+ * Copyright 2020 ConsenSys AG.
```

Comment:
Should we be updating the data every time we change the file and it's not the current year? I thought the date was only the original date the file was created but not entirely sure.

Reply:
Good spot.  Need to review the spotless header file.